packages/evm-adapter/src/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { ContractAddresses } from './constants';
+
+export const getContractAddress = (chainId: number): string | undefined => {
+  switch (chainId) {
+    case 80001:
```

Comment:
I think making an enum for those ids would make it easier to identify

Reply:
Ok